extensions/wikia/AdEngine/js/provider/gpt/helper.js

```diff
@@ -110,7 +86,7 @@ define('ext.wikia.adEngine.provider.gpt.helper', [
 
 		function callSuccess(adInfo) {
 			if (adInfo && adInfo.adType === 'collapse') {
-				slotTweaker.hide(element.getSlotName(), isBlocking());
+				slotTweaker.hide(element.getSlotName(), recoveryHelper.isBlocking());
```

Comment:
`recoveryHelper.isBlocking() && recoveryHelper. isRecoveryEnabled()`, otherwise it will have an impact to users from `wgAdDriverSorucePointDetection` countries